docker-compose.regtest.yml

```diff
@@ -54,10 +54,6 @@ services:
   rabbitmq:
     image: rabbitmq:3-management
     hostname: rabbitmq
-    environment:
```

Comment:
why rmoving this?

Reply:
Actually when I tried to run rabbitmq via docker-compose, it was complaining that 
rabbitmq_1          | error: RABBITMQ_DEFAULT_PASS is set but deprecated
rabbitmq_1          | error: RABBITMQ_DEFAULT_USER is set but deprecated
rabbitmq_1          | error: RABBITMQ_DEFAULT_VHOST is set but deprecated
rabbitmq_1          | error: deprecated environment variables detected
rabbitmq_1          | 
rabbitmq_1          | Please use a configuration file instead; visit https://www.rabbitmq.com/configure.html to learn more


Also I found that even if we do not provide any default ones, according to https://www.rabbitmq.com/access-control.html#default-state, It still uses same values.